src/java.desktop/share/native/libfontmanager/freetypeScaler.c

```diff
@@ -1285,10 +1285,12 @@ static int allocateSpaceForGP(GPData* gpdata, int npoints, int ncontours) {
     }
 
     /* failure if any of mallocs failed */
-    if (gpdata->pointTypes == NULL ||  gpdata->pointCoords == NULL)
+    if (gpdata->pointTypes == NULL || gpdata->pointCoords == NULL) {
+        if (gpdata->pointTypes != NULL)  { free(gpdata->pointTypes); gpdata->pointTypes = NULL; }
```

Comment:
You might want to add an extra space before `gpdata->pointTypes = NULL;` to align the statements vertically. You call.

Reply:
Can we also style it consistently, like:

```
  if (gpdata->pointTypes != NULL)  { 
      free(gpdata->pointTypes);
      gpdata->pointTypes = NULL;
  }
  if (gpdata->pointCoords != NULL) {
      free(gpdata->pointCoords);
      gpdata->pointCoords = NULL;
  }
```